roles/variables/defaults/main.yml

```diff
@@ -140,7 +140,7 @@ confluent_cli_base_path: /opt/confluent-cli
 confluent_cli_path: "/usr/local/bin/confluent"
 
 ### Confluent CLI version to download (e.g. "1.9.0"). Support matrix https://docs.confluent.io/platform/current/installation/versions-interoperability.html#confluent-cli
-confluent_cli_version: 1.43.0
+confluent_cli_version: 2.3.1
```

Comment:
```suggestion
confluent_cli_version: "{{ 2.3.1 if rbac_enabled else 1.43.0 }}"
```

Reply:
I answered in other comment, why we should not be doing this change.